### PR TITLE
Create HTML Character Sheet, Updated SAMPLE bestiary and Powers, Minor Fixes

### PR DESCRIPTION
**Pull request by LockerM**
Can ignore commits from before Jan 2 (bringing main branch back to present with this PR).

What was accomplished in scope of this PR:
- [X] Proposed 14 Powers for Powers SAMPLE
- [X] Updated Bestiary SAMPLE 
- [X] Minor fixes across bestiary and Powers lists 
- [ ] Initial Items YAML file

### CODE REVIEW

File: Clubs_level_1_char_sheet.html [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+
+<table style="float: left; font-size: 12pt">
+    <thead>
+    <tr>
+    
+      <th>Agility</th>
+    
+      <th>Conviction</th>
+    
+      <th>intelligence</th>
```
Comment (CBroz1) [inline Clubs_level_1_char_sheet.html:123]:
```suggestion
      <th>Intelligence</th>
```

File: Clubs_level_1_char_sheet.html [hunk 2]
```diff
@@ -0,0 +1,351 @@
+
+<html>
```
Comment (CBroz1) [inline Clubs_level_1_char_sheet.html:2]: The embedding on this file might be easier to see if we ran `prettier` on it. Like `black` but for other file types

File: automation/_input/04_Powers_SAMPLE.yaml [hunk 3]
```diff
@@ -95,6 +95,114 @@ Momentum: # Without this power, this should cost a major action.
         Role: Defender
         Skill: Brute > 1
 
+Attack, Mystic:
```
Comment (CBroz1) [inline automation/_input/04_Powers_SAMPLE.yaml:98]: What motivated including these in the sample?

File: automation/simulator/deck.py [hunk 4]
```diff
@@ -187,3 +187,34 @@ def check(self, TC: Card, TR: int, mod: int = 0):
                 result += "Color "
             result += "Hit" if draw.val in TC.range(TR) else "Miss"
         logger.info(f"Drew {draw} vs {TC} with TR {TR}: {result}")
+
```
Comment (CBroz1) [inline automation/simulator/deck.py:190]: Please don't add things I send you in chat unless you've changed it in some way. If we merge the changes here, I then need to resolve conflicts on my fork

File: automation/templates/character_sheets_html_v2.html [hunk 5]
```diff
@@ -0,0 +1,308 @@
+<!--use ctrl+shift+v to preview in vscode-->
+<htmL>
```
Comment (CBroz1) [inline automation/templates/character_sheets_html_v2.html:2]: Same deal with `prettier` - I think there's a vscode extension for that

File: Clubs_level_1_char_sheet.html [hunk 6]
```diff
@@ -0,0 +1,351 @@
...
+
```
Comment (CBroz1) [inline Clubs_level_1_char_sheet.html:3]: Why is this in the templates folder? I've been using that for code that is the template, not examples. Is it an input? is it an example output?

File: char_sheet_html.py [hunk 7]
```diff
@@ -0,0 +1,311 @@
+import jinja2 
+# import yaml
```
Comment (CBroz1) [inline char_sheet_html.py:2]: In general, I avoid merging commented code to main

File: char_sheet_html.py [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+</div>
+
+<br style="clear: both;">
+
+</body>
+</htmL>
+""")
+
+# render() command requires each variable be called within the template to function correctly
+html = template.render(char_name=char_name, char_pronouns=char_pronouns, char_role=char_role, char_level=char_level, powers=powers, vulnys=vulnys, items=items, attributes=attributes, skills=skills, stats=stats)
```
Comment (CBroz1) [inline char_sheet_html.py:306]: please install the `black` formatter. lmk if you want some help setting it up so that it auto-formats on save

File: docs/src/1_Mechanics/06_Bestiary.yaml [hunk 9]
```diff
@@ -295,17 +310,59 @@ Ritualist, Support:
...
+  House Actions:
+  - In House:
+      - Summon: Summon 2x minions
+      - Entangle: Entangle 2x targets
+      - Mind control: 2 uses, only if 'primed' - grappled or entangled
+  - In Lake:  
+      - Clubs: Lake attempts to grab every creature currently in the lake. DR 2 save, on failure entangled and dragged 5 feet under water.
+      - Diamonds: Fissure appears in the ground in a determined quadrant. DR 2 save, on failure entangled. 
+      - Spades: Force push on a space determined by GM. Pushed 5 spaces back from push, DR 2 save, on failure take 1 damage. 
+      - Hearts: Sugar crystals spray from the lake, targeting anyone within 2 spaces of the lake shore. DR 2 save, on failure take 2 damage. On success, half damage.
```
Comment (CBroz1) [inline docs/src/1_Mechanics/06_Bestiary.yaml:343]: We had previously discussed House Actions as one type of Power exclusive to Bosses. Separating them from powers here implies a different organization structure. 

Similarly, teleport minions above is not implemented as a power

File: Clubs_level_1_char_sheet.html (hunk 6 above)
Comment (LockerM) [reply Clubs_level_1_char_sheet.html:3]: This is an example Output, not sure where that should go. Happy to move or adjust accordingly (can remove entirely) depending on how we want structure of this to go.

File: automation/_input/04_Powers_SAMPLE.yaml (hunk 3 above)
Comment (LockerM) [reply automation/_input/04_Powers_SAMPLE.yaml:98]: Currently only Defender and Martial are supported in the SAMPLE, adding additional powers gives a fuller representation of Powers a Support and Caster could use.

File: automation/simulator/deck.py (hunk 4 above)
Comment (LockerM) [reply automation/simulator/deck.py:190]: Sorry, I can attempt to remove these from my file and re-commit to remove them from the PR.

File: docs/src/1_Mechanics/06_Bestiary.yaml (hunk 9 above)
Comment (D-D-2) [reply docs/src/1_Mechanics/06_Bestiary.yaml:343]: As long the end user sees House Actions as separate from Powers. Does that complicate the code to want it to print as separate boxes but write it as the same type?

File: automation/_input/04_Powers_SAMPLE.yaml (hunk 3 above)
Comment (CBroz1) [reply automation/_input/04_Powers_SAMPLE.yaml:98]: I propose we reserve the sample for items that test the bounds of what the code can handle, edge cases. That might include items that shouldn't actually be in the game (e.g., what if a power had a save and a stat override)

File: automation/simulator/deck.py (hunk 4 above)
Comment (CBroz1) [reply automation/simulator/deck.py]: thanks!

File: Clubs_level_1_char_sheet.html (hunk 6 above)
Comment (CBroz1) [reply Clubs_level_1_char_sheet.html]: I can work on a proposal for how to organize this kind of export

File: docs/src/1_Mechanics/06_Bestiary.yaml (hunk 9 above)
Comment (CBroz1) [reply docs/src/1_Mechanics/06_Bestiary.yaml:343]: There's the organization presented to outside viewers (e.g., Powers = major+minor+passive, separate from Vulnys, etc) and the organization we use internally (e.g., Powers are major+minor+vulnys, etc). In general, it's best to align these as it simplifies code, but it's not hard to pull a subset of a list based on a type

File: docs/src/1_Mechanics/07_Items.yaml [hunk 10]
```diff
@@ -0,0 +1,180 @@
+Template:               # must be unique
```
Comment (CBroz1) [inline docs/src/1_Mechanics/07_Items.yaml:1]: Let's try to simplify down to exactly what we need for now. We could work on a sample version to see if that works when loaded as part of a PC render. A StatOverride feature could work here like i've implemented for Powers

**Review by CBroz1 — CHANGES_REQUESTED** [review]
